src/Engine/Texture/TextureAsset.cpp

```diff
@@ -34,6 +35,9 @@ void TextureAsset::Load(const std::string& name) {
     
     // Get properties from meta file.
     Json::Value root;
+    if(!FileSystem::FileExists(std::string(filename + ".json").c_str())) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'if' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)